packages/x-internals/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-internals",
-  "version": "8.0.0-alpha.2",
+  "version": "8.0.0-alpha.3",
```

Comment:
```suggestion
  "version": "8.0.0-alpha.5",
```

Reply:
Are we ignoring what Lerna suggests? 👀

Reply:
Yes, we always sync package versions when we do a release.
Lerna increments the existing version by default. :)